pkg/minikube/assets/addons.go

```diff
@@ -95,9 +107,11 @@ var Addons = map[string]*Addon{
 
 		//GuestPersistentDir
 	}, false, "auto-pause", map[string]string{
-		"haproxy": "haproxy:2.3.5",
+		"haproxy":       "haproxy:2.3.5",
+		"AutoPauseHook": "azhao155/auto-pause-hook:1.7",
 	}, map[string]string{
-		"haproxy": "gcr.io",
+		"haproxy":       "gcr.io",
```

Comment:
just noticed this, was this ever correct ? I think haproxy image is in docker hub

Reply:
it's not needed since we are not using template in haproxy.